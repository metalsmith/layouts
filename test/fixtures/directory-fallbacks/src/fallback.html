---
layout: base.html
---
<p>Content</p>
