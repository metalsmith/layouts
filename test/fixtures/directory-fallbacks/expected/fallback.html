<header>Head</header>
<p>Nav</p>
<p>Content</p>

<footer>Footer</footer>
